### PR TITLE
Occitan version

### PR DESCRIPTION
**Pull request by Quent-in**
| Q             | A
| ------------- | ---
| Translation   | yes
| Fixed tickets | comma-separated list of tickets fixed by the PR, if any
| License       | MIT

I hope everything is fine now! Looking forward to seeing it running :)

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+        about: 'A prepaus'
+    page_title: 'Configuracion'
+
+config:
+    tab_menu:
+        settings: 'Paramètres'
+        rss: 'RSS'
+        user_info: 'Mon compte'
+        password: 'Senhal'
+        rules: 'Règlas d\'etiquetas automaticas'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml]: You should remove simple quote around and put double quote:

> rules: "Règlas d'etiquetas automaticas"

It also apply to the 3 others occurence in this file. You should fix them.

Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml]: Can you remove the `d\'` part, and put the string into double quotes : `"Règlas d'etiquetas automaticas"`

**Comment by Quent-in** [comment]
Is it ok now?

**Comment by j0k3r** [comment]
You should update your file and push changes on the same branch not by creating a new one :)

**Comment by Quent-in** [comment]
I pushed "edit" I didnt except it would creat something else. Can do it for me, I start to lose temper with GitHub logic :(

**Comment by j0k3r** [comment]
So I fixed bugs manually here https://github.com/wallabag/wallabag/commit/1695a6fecd517a1324522fd74bb39c52aa9271a1
Thanks for that contribution !

**Pull request merged**